src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -1,58 +1,209 @@
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
 ```
 
 </Block>
 </BlockSwitcher>
 
-## Install Amplify libraries
+> Amplify CLI will infer the proper configuration based on the type of project Amplify is being initialized in where possible. Here it detected Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
 
-Next, install the local Amplify dependencies. The directions here will depend on whether you are using Expo or the React Native CLI.
+When you initialize a new Amplify project, a few things happen:
 
-import all0 from "/src/fragments/start/getting-started/reactnative/getting-started-steps-ui.mdx";
+- It creates a top level directory called **amplify** that stores your backend definition. During the tutorial you'll add capabilities such as a GraphQL API and authentication. As you add features, the **amplify** folder will grow with infrastructure-as-code templates that define your backend stack. Infrastructure-as-code is a best practice way to create a replicable backend stack.
+- It creates a file called **aws-exports.js** in the **src** directory that holds all the configuration for the services you create with Amplify. This is how the Amplify client is able to get the necessary information about your backend services.
+- It modifies the **.gitignore** file, adding some generated files to the ignore list
+- A cloud project is created for you in the AWS Amplify Console that can be accessed by running `amplify console`. The Console provides a list of backend environments, deep links to provisioned resources per Amplify category, status of recent deployments, and instructions on how to promote, clone, pull, and delete backend resources
 
-<Fragments fragments={{all: all0}} />
+## Install Amplify Libraries
 
-Now your project is set up and you can begin adding new features.
+The `aws-amplify` package is the main library for working with Amplify Libraries in your projects:
 
-## Run your app
+```bash
+npm install aws-amplify
+```
+
+## Set up frontend
+
+Next, configure Amplify so it can interact with backend services.
+
+Open **index.js** and add the following code below the last import:
 
-To run your React Native app, you can follow the below steps
+```javascript
+import { Amplify } from 'aws-amplify';
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+```
 
-import reactnative1 from '/src/fragments/lib/project-setup/react-native/run-react-native.mdx';
+And that's all it takes to configure Amplify. As you add or remove categories and make updates to your backend configuration using the Amplify CLI, the configuration in **aws-exports.js** will update automatically.
 
-<Fragments fragments={{ "react-native": reactnative1 }} />
+Now that your React app is set up and Amplify is initialized, you can add an API in the next step.
```

Comment:
This line might also be inaccurate given the different tutorials

Reply:
One last thing. I see a lot of numbered fragment variables. It's very confusing and hard to maintain. Is there any reason for this (some kind of automatization or indexing)?

```jsx
import react1 from "/src/fragments/start/getting-started/react/api.mdx";

<Fragments fragments={{react: react1}} />

import angular2 from "/src/fragments/start/getting-started/angular/data-model.mdx";

<Fragments fragments={{angular: angular2}} />
```

I find this much easier to read:

```jsx
import reactApi from "/src/fragments/start/getting-started/react/api.mdx";

<Fragments fragments={{react: reactApi}} />

import angularDataModel from "/src/fragments/start/getting-started/angular/data-model.mdx";

<Fragments fragments={{angular: angularDataModel}} />
```

Reply:
I checked with team members and it appears to have been the result of some refactoring work from the past which necessitated an automation script which simply incremented these fragment references. I do agree that have a more verbose and meaningful name is more useful and since there is not a reason to keep the existing pattern, I agree with you we should opt for readability 👍